Source/DafnyTestGeneration/TestMethod.cs

```diff
@@ -275,6 +275,26 @@ public class TestMethod {
             elements.Add(ExtractVariable(element, asBasicSeqType?.TypeArgs?.FirstOrDefault((Type?)null)));
           }
           seqName = "d" + nextValueId++;
+
+          //
+          // Hack to work around stack overflow issue in Dafny construction of large strings.
+          // See https://amzn-aws.slack.com/archives/C018E843J0H/p1657839224937449
+          //
+          if (elements.Count > 200) {
+            int i = 0;
+            string outstr = "";
+            int chunksize = 100;
+            List<string> chunkStrs = new List<string>();
+            while (i < elements.Count) {
+              int count = Math.Min(chunksize, elements.Count - i);
+              string chunk = "\"" + string.Join("", elements.GetRange(i, count)).Replace("'", "") + "\"";
+              chunkStrs.Add(chunk);
+              i += chunksize;
+            }
+            return string.Join("+", chunkStrs);
+          }
```

Comment:
Ah, i think it might fix a bug I was getting for AWS Encryption SDK Tests!

Reply:
Yes, another semi-hack that I needed to get the symbolic tester demo running. At its core this change is probably OK to include, but yeah, we'll need to make sure it works in all cases.